bin/create-config

```diff
@@ -14,9 +15,33 @@ parser.add_argument('source', metavar='SOURCE-SHAPEFILE-PATH')
 parser.add_argument('boundary', metavar='NAME-OF_BOUNDARY')
 
 args = parser.parse_args()
+boundary_path = Path('./boundaries/build') / Path(args.boundary)
+source_path = Path(args.source)
 
+# Create a folder
 try:
-    boundary_path = Path('./boundaries/build') / Path(args.boundary)
     boundary_path.mkdir()
 except FileExistsError as e:
     sys.exit('{} already exists. Please pick a new name'.format(args.boundary))
+
+# Get existing json entries
+with open('./boundaries/boundary-config.json', 'r') as jsonFile:
+    try:
+        config_entries = json.load(jsonFile)
+    except json.decoder.JSONDecodeError as e:
+        config_entries = []
+
+# Make a new entry
+src_proj4 = proj4_from_shp(str(source_path))
+src_encoding = encoding_from_shp(str(source_path))
+new_config = BoundaryConfig(shp_path=str(source_path),
+                            proj4=src_proj4,
+                            encoding=src_encoding,
+                            boundary=args.boundary)
+
+# Append it to existing entries
+config_entries.append(new_config.dumps())
+
+# Write all entries back to file..
+with open('./boundaries/boundary-config.json', 'w') as jsonFile:
```

Comment:
A tiny stylistic thing, but I'd try to keep all variable names as snake case, rather than lower camel case.